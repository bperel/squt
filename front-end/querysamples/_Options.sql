SELECT DISTINCT STRAIGHT_JOIN 1;