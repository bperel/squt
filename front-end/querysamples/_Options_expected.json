{
	"Constants" : {
		"0" : {
			"alias" : "1",
			"value" : "1",
			"to" : "OUTPUT"
		}
	},
	"Options" : {
		"SELECT_DISTINCT" : 1,
		"SELECT_STRAIGHT_JOIN" : 1
	}
}